util/syncpacketid_derive/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "syncpacketid_derive"
```

Comment:
can you make this crate `rust 2018` and remove `extern crate`?

Reply:
Sure, I'll do that.